packages/init/__tests__/__stubs__/fake-project/documents/en-us/Xs5vWREAACYAIvvr=#=Xs5vWREAACYAIvvs=#=top_menu=#=Xs5vWREAACYAIvvt=#=en-us=#=y.json

```diff
@@ -1,57 +0,0 @@
-{
```

Comment:
Are stubs not being used any more?

Reply:
We use `@prismicio/mock` to generate test fixtures. It lets us configure the generated data and introduce some randomness into tests.

More details are shared here: https://github.com/prismicio/slice-machine/pull/799#discussion_r1115140191